consumer/dragonboard/qualcomm-robotics-rb1/installation/le.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Linux Host Installation for Qualcomm Robotics RB1 development kit
+permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb1/installation/le.md.html
+---
+## Linux Host
+
+This section show how to install a new operating system to your Qualcomm Robotics RB1 development kit using the fastboot method on a Linux host computer.
+
+***
+
+- **Step 1**: Make sure fastboot is set up on host computer
+- **Step 2**: Connect host computer to Qualcomm Robotics RB1 dev kit
+- **Step 3**: Boot RB1 into fastboot mode
+- **Step 4**: Flash Images
+- **Step 5**: Reboot the development kit
+
+***
+
+#### **Step 1**: Make sure fastboot is set up on host computer.
+##### You can install fastboot from source or using your distro package manager.
+
+- **From source:**
+  - Android SDK “Tools only” for Linux can be downloaded <a href="https://developer.android.com/studio/releases/platform-tools.html" target="_blank">here</a>
+  - The Linux “Tools Only” SDK download does not come with fastboot, you will need to use the Android SDK Manager to install platform-tools.
+  - To do this follow the “SDK Readme.txt” instructions included in your SDK “Tools Only” download.
+
+  If you are still having trouble setting up fastboot, <a href="https://youtu.be/W_zlydVBftA" target="_blank">click here</a> for a short tutorial video
+
+- **Or using a distro package manager:**
+  ```
+  # Red Hat flavour (Fedora, CentOS)
+  $ sudo yum install android-tools
+
+  # Debian (Ubuntu)
+  $ sudo apt-get install google-android-platform-tools-installer
```

Comment:
This package is not available on ubuntu,

Reply:
@Mani-Sadhasivam 
Here I referred to https://packages.ubuntu.com/kinetic/google-android-platform-tools-installer , is this package available on Ubuntu?

Reply:
Only since 22.10. It might be easier to install just the `fastboot` package. At least it is available in all supported Debian versions and in Ubuntu since 20.04 LTS

Reply:
I've checked the latest Ubuntu LTS releases (xenial, bionic, focal, jammy), and all of them provide fastboot by installing a `fastboot` package, thus I'll replace google-`android-platform-tools-installer` with `fastboot`.